lib/pharos/configuration/host.rb

```diff
@@ -145,7 +145,7 @@ def kubelet_args(local_only: false, cloud_provider: nil)
         if crio?
           args << '--container-runtime=remote'
           args << '--runtime-request-timeout=15m'
-          args << '--container-runtime-endpoint=unix:///var/run/crio/crio.sock'
+          args << '--container-runtime-endpoint=/var/run/crio/crio.sock'
```

Comment:
Maybe add some comment & link to upstream issue why the path need to be without the prefix

Reply:
Done.